code/modules/power/smes.dm

```diff
@@ -65,6 +65,10 @@
 	var/datum/effect_system/sparks/big_spark
 	var/datum/effect_system/sparks/small_spark
 
+	var/time_depleted = 0
+	var/Queue/output_history = new /Queue()

```

Comment:
Isn't it standard to but this sort of thing in Initialize or new?

Reply:
Yeah, I moved it to Initialize. I just did not push my commits. I have to fix some other things before I push it.